apps/src/lib/levelbuilder/lesson-editor/ActivityCard.jsx

```diff
@@ -139,7 +139,9 @@ class ActivityCard extends Component {
     this.props.updateActivityField(
       this.props.activity.position,
       'duration',
-      event.target.value
+      Number.isNaN(parseInt(event.target.value))
+        ? ''
+        : parseInt(event.target.value)
```

Comment:
I would suggest using `undefined` or `null` to represent the absence of a value here. the rest of my inline comments are only relevant if you decide to do this...

Reply:
This is why I went with the empty string

<img width="791" alt="Screen Shot 2020-10-28 at 9 27 23 PM" src="https://user-images.githubusercontent.com/208083/97514118-606b4980-1964-11eb-9305-81d1f8dd2149.png">

Reply:
ok, makes sense!